sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/ResourceCollectionUtils.java

```diff
@@ -54,10 +54,56 @@
                 .collect(Collectors.toList());
     }
 
+    public static Optional<String> getRole(Resource resource) {
```

Comment:
this class name is already falling apart -- lots of functions that aren't operating on collections of resources. back to `ResourceUtils`?